cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -330,3 +332,28 @@ func (c Client) WaitForClusterAvailable(delay time.Duration, retries int, retryI
 func CheckConfigurationIsHA(cfg *kubeadmapi.Etcd) bool {
 	return cfg.External != nil && len(cfg.External.Endpoints) > 1
 }
+
+// ipPortURL creates an HTTPS URL based on an IP address
+// and  a port. The URL that is created is of the form:
+//     https://<ip>:<port>
+func ipPortURL(host string, port int) string {
```

Comment:
I don't think this really needs to live here, but I don't think it's blocking for me. 

/approve 
/assign @kad   
/cc @bart0sh

Reply:
I'd do simpler `"https://" + net.JoinHostPort(...)` instead of calling that function, but this not critical.

Reply:
@leblancd please just remove the utility as per the above comments then LGTM.
thanks!